### DISCUSSION TITLE
buggy documentation

### DISCUSSION THREAD

**Discussion by alexanderstr** [deepgram/community#886] [General help] [answered]
I tried to use the documentation for my first integration, and the very first code sample I tried was in v3, but the assembly is 4.1.

I found a guide on how to migrate (why not just migrate your documentation?), and the guide has numerous typos in the very first example.

For example, it is "PreRecordedSchema," not "PrerecordedSchema."

![bug](https://github.com/user-attachments/assets/ff500f81-57d9-470b-8ea6-64b90e701011)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
hi @alexanderstr 

Thanks for pointing that out. Will give this an update.

EDIT: It should be updated already. Flush out the browser cache in a couple of minutes.

**Reply by alexanderstr** [reply]
any chance to move the documentation to v4 please ? the changes are significant enough that it takes effort to look at 2 points of reference.

**Reply by davidvonthenen** [reply]
Hi @alexanderstr 

Not sure what you mean. Are you not seeing this:

![Screenshot 2024-08-07 at 14 18 41](https://github.com/user-attachments/assets/3fcf9f12-cc67-49a0-ba89-14dd24f27dc8)

**Reply by alexanderstr** [reply]
ha! i did not realize that v4 was there. thank you!

**Reply by davidvonthenen** [reply]
No worries. I have found ReadMe can be unpredictable sometimes when you save docs. I have seen weirder things.
